smqtk_image_io/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .interfaces.image_reader import ImageReader
+from .bbox import AxisAlignedBoundingBox
```

Comment:
Check out the __version__ set in the other smqtk-* packages: we want to keep that attribute here but can draw from pkg_resources for the package version now.